src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -19,8 +19,8 @@ public partial class ConfigurationClient
         const string ComponentVersion = "1.0.0";
 
         static readonly HttpPipelinePolicy s_defaultRetryPolicy = RetryPolicy.CreateFixed(3, TimeSpan.Zero,
-            //429, // Too Many Requests TODO (pri 2): this needs to throttle based on x-ms-retry-after 
-            500, // Internal Server Error 
+            //429, // Too Many Requests TODO (pri 2): this needs to throttle based on x-ms-retry-after
```

Comment:
nit:  We should consider honoring the standard HTTP [`Retry After`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) header as well.

Not in the scope of your changes, but it may be worth annotating while there are changes being made.